docs/content/how-to-guides/configure-reactions/configure-drasi-result-reaction/_index.md

```diff
@@ -3,8 +3,137 @@ type: "docs"
+
+## Reaction Definitions
+The YAML file passed to `drasi apply` can contain one or more Reaction definitions. Here is an example of a Drasi Debug Reaction definition:
+
+```yaml {#quick-result-reaction}
+kind: Reaction
+apiVersion: v1
+name: quick-result-reaction
+spec:
+  kind: Result
+  queries:
+    query1:
+  endpoints:
+    gateway: 8080  
+```
+
+In this definition: 
+- the **apiVersion** must be **v1**.
+- the **kind** property tells Drasi to create a **Reaction** resource.
+- the **spec.kind** property tells Drasi the kind of Reaction to create, in this case a **Result** Reaction. 
+- the **name** property tells Drasi the identity of the Reaction and must be unique within the scope of Reactions within the target Drasi environment. In the above example, the **name** of the Reaction is **quick-result-reaction**.
+
+This table describes the other settings in the **spec** section of the Reaction definition:
+|Property|Description|
+|-|-|
+|queries|Specifies the set of **names** of the Continuous Queries the Reaction will subscribe to.|
+|endpoints.gateway|Specifies the **port** on which the Drasi Result Reaction operates. If not specified, this defaults to 8080. |
+
+## Inspecting the Reaction
+As soon as the Reaction is created it will start running, subscribing to the specified list of Continuous Queries and processing changes to the Continuous Query results.
+
+You can check the status of the Reaction using the `drasi list` command:
+
+```text
+drasi list reaction
+```
+
+Or including a target namespace:
+
+```text
+drasi list reaction -n drasi-namespace
+```
+
+This will return a simple list of all Reactions in the default (or specified) namespace and their overall status. For example:
+
+```
+        ID              | AVAILABLE
+------------------------+------------
+  quick-result-reaction | true
+```
+
+If an error has occurred during the creation or operation of a Reaction, the `AVAILABLE` column will contain the error text instead of `true` or `false`.
+
+For more details about the Reaction you can use the [drasi describe](/reference/command-line-interface#drasi-describe) command:
+
+```text
+drasi describe reaction quick-result-reaction
+```
+
+This will return the full definition used to create the Reaction along with more detailed status information.
+
+
+## Using the Result Reaction
+Because the Drasi Debug Reaction is running inside a Kubernetes cluster, you need to enable access to the port through which you can send HTTP requests. The easiest way to do this is to setup a port forward using `kubectl` and the following command:
+
+```bash
+kubectl port-forward -n <drasi-namespace> services/quick-result-reaction-gateway 8080:8080
```

Comment:
I think we need to say something about exposing this service to the outside world for production purposes?

Reply:
I have added this part before the command:

The only way to retrieve the result set from the Result Reaction is through sending HTTP GET requests to the endpoints. For development and testing purposes, since the Result Reaction lives in a Kubernetes pod, it needs to be exposed using Kubernetes port-forwarding. This allows you to send HTTP GET requests to the Result Reaction endpoint from your local machine or from another application. Use the following command to set up port-forwarding for the Result Reaction deployed in the previous steps:

Reply:
Should we also mention that it is possible to create an ingress for production scenarios?